docs/source/hardware/lowpriorityFAQ.rst

```diff
@@ -0,0 +1,259 @@
+.. _partition_names:
+
+**********
+Low priority partition FAQs
+**********
+:sub:`Updated November 2022`
+
+Partition information
+===================
+
+**1. What is the** ``lowpriority`` **partition and why is it important?**
+
+``lowpriority`` is a new partition on Discovery that allows the research community to use **private resources** on 
+the cluster when they are idle. This new partition has hardware not otherwise available to the general research 
+community and, in time, could double the resources available to NURC users. This is a common practice in HPC clusters 
+to optimize the use of idle private resources that consume power and cooling. 
+
+More information about the low priority partition and examples on how to use it can be 
+found `here <https://northeastern-university-rc-public-documentation--19.com.readthedocs.build/en/19/hardware/lowpriority.html>`_. 
+
+**2. What is a "low priority" job?**
+
+A job submitted to the ``lowpriority`` partition that comes from a user **who is NOT** a member of the private 
+partition on which the job is scheduled to run.
+
+**3. What is a “high priority” job?**
+
+A job submitted to a private partition by a user **who is** a member of that private partition, on which a low 
+priority job is running.
+
+**4. How does SLURM handle low priority jobs?**
+
+In SLURM, jobs are submitted to partitions. SLURM is configured such that jobs submitted to the ``lowpriority`` 
+partition are considered for execution on private partitions only when they are idle. 
+
+In other words, a low priority job will be executed only if the private partition has resources to execute a 
+job submitted to it. However, if a member of the private partition submits a job during this time, then the low 
+priority job will be automatically stopped/suspended and re-queued (preempted). 
+
+Jobs submitted to the ``lowpriority`` partition have the lowest priority of any class of jobs and are only considered 
+after all other normal/high priority jobs have been placed. Jobs submitted by private partition members to their own 
+partition always have higher priority. 
+
+**5. Will the** ``lowpriority`` **partition become the default partition?**
+
+No, the default partition will still be partition `short`. By default, jobs are not subject to preemption.
```

Comment:
Did you mean "By default, jobs **on the short partition** are not subject to preemption."?

Reply:
@marianalevineu I think this intends to say that short is the default partition and not lowpriority. If lowpriority becomes the default partition, that would also mean that all jobs are subject to preemption by default, which is not the case.

Reply:
I'll change the second sentence to "This ensures that by default, jobs are not subject to preemption" if that reads better.

Reply:
thanks for the clarification @ShobanaSekar , feel free to resolve this conversation once the text it changed.